tests/ui/UnreadIndicatorsTest.js

```diff
@@ -184,8 +200,11 @@ function signInAndGetAppWithUnreadChat() {
         .then(() => renderedApp);
 }
 
-describe.skip('Unread Indicators', () => {
-    afterEach(() => Onyx.clear());
+describe('Unread Indicators', () => {
+    afterEach(() => {
+        jest.clearAllMocks();
```

Comment:
I'm curious why this was necessary

Reply:
This should make sure mock function call counts are reset after every test. So we'll be correctly validating that `LocalNotification.showCommentNotification` was called during the notification test (and not from another test)